projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-details/replenishment-order-cancellation/replenishment-order-cancellation.component.html

```diff
@@ -15,7 +15,7 @@
     *ngIf="(replenishmentOrder$ | async).active"
     class="col-xs-12 col-md-4 col-lg-3"
   >
-    <button class="btn btn-block btn-action" (click)="openDialog()">
+    <button #element class="btn btn-block btn-action" (click)="openDialog()">
```

Comment:
What is this id?

Reply:
it was for the view child, which wasn't hooked up before in the component (the one similar to anonymous)